### ISSUE TITLE
github-actions bot does not remove stale-label after issue is commented on

### ISSUE THREAD

**Issue by Jonta** [laurent22/joplin#5988]
Example issues:

https://github.com/laurent22/joplin/issues/4672
https://github.com/laurent22/joplin/issues/4685
https://github.com/laurent22/joplin/issues/4671

If @github-actions takes significantly longer to do so than @stale used to do, I suggest updating its message to reflect that. :)

**Comment by Jonta** [comment]
They're removed now, but it still took about 3 hours for that to happen. Why is that? :)

**Comment by Jonta** [comment]
@github-actions Oh you

**Comment by Jonta** [comment]
@roman-r-m Wanna tag this with `question-w4a`? =)

**Comment by Daeraxa** [comment]
> github-actions bot does not remove stale-label after issue is commented on

![image](https://user-images.githubusercontent.com/58074586/153507465-52b313d7-d307-4b8d-8b2c-b93c90b4db72.png)

**Comment by Jonta** [comment]
@Daeraxa Yes. Eventually. I think a 5 hour response time is not acceptable. It used to be just a few seconds

**Comment by Daeraxa** [comment]
I don't see why that is important. It still does what it says.

**Comment by Jonta** [comment]
5 hours instead of 5 seconds is bad UX

I think people's mental model of a bot like this is similar to that of getting a sign-up-confirmation email

A couple of minutes might be acceptable

**Comment by Daeraxa** [comment]
In which case it is an enhancement/feature request and not a bug, therefore not suitable for the issue tracker.

**Comment by Jonta** [comment]
Where this project draws the line between enhancement and bug is up to this project. =)

If this stays on the order of hours, I predict it won't be the last time such an issue is opened, and therefore suggest keeping this one in this issue tracker, as a reference point

**Comment by Daeraxa** [comment]
I think the line is pretty clear, a bug is an unintended behaviour. 
Its a bot that runs its actions once per day, I really don't think that it is confusing nor does it really matter what the UX is, the issues log is a tool for the maintainers, not the user.

**Comment by Jonta** [comment]
@Daeraxa 

> Its a bot that runs its actions once per day

Where is this communicated to people like me?

> I really don't think that it is confusing

But I do, and my money's on other people finding this transition confusing as well

> nor does it really matter what the UX is

How so?

> the issues log is a tool for the maintainers, not the user.

How so? A user doesn't have to report an issue that has already been reported. Is it a tool used more by maintainers than users? Sure. But not _only_
